pkgs/by-name/ar/arti/package.nix

```diff
@@ -28,7 +28,7 @@ rustPlatform.buildRustPackage rec {
 
   buildInputs =
     [ sqlite ]
-    ++ lib.optionals stdenv.isLinux [ openssl ]
```

Comment:
These sneaked back in https://github.com/NixOS/nixpkgs/commit/5136e375f2e764f7d11ec87455ca05498f723af8 lol